framework/test/jest/unit/specs/modules/chain/dpos/getRoundDelegates/getRoundDelegates.spec.js

```diff
@@ -81,20 +64,13 @@ describe('dpos.getRoundDelegates()', () => {
 			list = await dpos.getRoundDelegates(roundNo);
 		});
 
-		it('should return shuffled delegate public keys by using round_delegates table', () => {
+		it('should return shuffled delegate public keys by using round_delegates table record', () => {
```

Comment:
Do we have a test for checking if the `create` method gets called if the delegateList.length === 0 ?

Reply:
Of course :D just try to see the raw file bitte.

Reply:
I tried :D It's horrible on Github haha